app/Http/Controllers/Apis/Protected/Summit/Factories/SponsorServicesStatisticsValidationRulesFactory.php

```
@@ -32,4 +32,15 @@ public static function buildForUpdate(array $payload = []): array
     {
         return self::buildForAdd();
     }
+
+    public static function buildForBulkUpdate(array $payload = []): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$payload'. (undefined)

(UnusedFormalParameter)